### ISSUE TITLE
Write a GTN Style Guide

### ISSUE THREAD

**Issue by hexylena** [galaxyproject/training-material#3935]
Including
- how to structure
  - how to e.g. tell people to run workflows (maybe it should just be a snippet though?)
- best practices
- tips level of detail to go into

**Comment by bebatut** [comment]
I am not sure to understand what you want here. A style guide for tutorial or something else?

**Comment by hexylena** [comment]
Yes! Exactly, for tutorials. Something an author can refer to if they're not sure how they should structure some action a user should take (e.g. tool or workflow or viz invocation), and that we reviewers can point to as examples of how to properly structure those bits.

Also e.g. digital ocean's tutorials are often seen as "consistently quite good", and I think a lot of that is achieved by having a good style and technical guide for authors.
https://www.digitalocean.com/community/tutorials/digitalocean-s-technical-writing-guidelines

**Comment by hexylena** [comment]
https://training.galaxyproject.org/training-material/topics/contributing/tutorials/create-new-tutorial/tutorial.html is very technical on generating skeletons (perhaps should even be renamed to "generate tutorial skeleton")
https://training.galaxyproject.org/training-material/topics/contributing/tutorials/create-new-tutorial-content/tutorial.html is great but also exceedingly technical on the markdown side.

I guess I'm thinking something more like DO's, linking to the markdown tutorial where relevant

**Comment by bebatut** [comment]
Sounds good :smile:
